src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -1263,6 +1263,11 @@ namespace cryptonote
     m_mempool.set_relayed(txs);
   }
   //-----------------------------------------------------------------------------------------------
+  crypto::hash core::get_block_id(const uint64_t height)
+  {
+    return get_block_id_by_height(height);
+  }
```

Comment:
That'll have to be added to a couple unit tests IIRC (or just use get_block_id_by_height ?)

Reply:
I tried to just use get_block_id_by_height but some other files failed to compile, trying to cast core to some other interface. Had no idea what the problem was and didn't dig deeper.

Reply:
moved away to blockchain.h